onnxruntime/core/mlas/lib/systolic/systolic_include.h

```diff
@@ -2072,18 +2074,19 @@ static void tiled_conv(
     const int pool_out_dim = (out_dim + 2*pool_padding - pool_size) / pool_stride + 1;
     const int dilated_in_dim = in_dim + (input_dilation-1)*(in_dim-1);
 
+    int per_thread_ochs = out_channels / NTHREADS;
+
     for (int b = 0; b < batch_size; b += batches) {
         for (int porow = 0; porow < pool_out_dim; porow += porows) {
             const int orow = porow * pool_stride - pool_padding;
 
             for (int pocol = 0; pocol < pool_out_dim; pocol += pocols) {
                 const int ocol = pocol * pool_stride - pool_padding;
-
-                for (int poch = 0; poch < out_channels; poch += pochs) {
+                printf("total_och: %d, per_thread_och: %d\n", out_channels, out_channels_thread);
```

Comment:
It's fine to have this for debugging, but can we remove this `printf` from the PR (or add a switch that users have to set to see this `printf`)?